[2025-01-26T22:06:03.744+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-26T22:06:03.793+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_processing.schema_create scheduled__2025-01-26T22:04:00+00:00 [queued]>
[2025-01-26T22:06:03.818+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_processing.schema_create scheduled__2025-01-26T22:04:00+00:00 [queued]>
[2025-01-26T22:06:03.818+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-26T22:06:03.857+0000] {taskinstance.py:2889} INFO - Executing <Task(SQLExecuteQueryOperator): schema_create> on 2025-01-26 22:04:00+00:00
[2025-01-26T22:06:03.879+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1643) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-26T22:06:03.883+0000] {standard_task_runner.py:72} INFO - Started process 1652 to run task
[2025-01-26T22:06:03.887+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_processing', 'schema_create', 'scheduled__2025-01-26T22:04:00+00:00', '--job-id', '414', '--raw', '--subdir', 'DAGS_FOLDER/DAG_etl.py', '--cfg-path', '/tmp/tmp_rm3zm32']
[2025-01-26T22:06:03.893+0000] {standard_task_runner.py:105} INFO - Job 414: Subtask schema_create
[2025-01-26T22:06:04.058+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_processing.schema_create scheduled__2025-01-26T22:04:00+00:00 [running]> on host a664bcf281bf
[2025-01-26T22:06:04.719+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_processing' AIRFLOW_CTX_TASK_ID='schema_create' AIRFLOW_CTX_EXECUTION_DATE='2025-01-26T22:04:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-26T22:04:00+00:00'
[2025-01-26T22:06:04.723+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-26T22:06:04.765+0000] {sql.py:278} INFO - Executing: 
    create schema if not exists nds;
    create schema if not exists dds;
    create schema if not exists stg;
[2025-01-26T22:06:04.789+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:06:04.807+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:06:04.815+0000] {sql.py:544} INFO - Running statement: 
    create schema if not exists nds;
    create schema if not exists dds;
    create schema if not exists stg;, parameters: None
[2025-01-26T22:06:04.846+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-26T22:06:04.847+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_processing, task_id=schema_create, run_id=scheduled__2025-01-26T22:04:00+00:00, execution_date=20250126T220400, start_date=20250126T220603, end_date=20250126T220604
[2025-01-26T22:06:04.919+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-26T22:06:05.356+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-26T22:06:05.372+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
